app/views/layouts/application.html.erb

```diff
@@ -3,7 +3,7 @@
   <head>
     <%= render partial: "layouts/head" %>
   </head>
-  <body class="app-thing">
+  <body class="app-tesseract">
```

Comment:
I agree `app-thing` was poorly named (clearly we extracted this gem from `thing` at some point lol`). However, I think maybe `app-theme` or `app-mitlibtheme` might be clearer later where the class is coming from.

Reply:
I agree with the need to choose this name intentionally; I don't object to either of those values, but one of the difficulties I've had when dealing with this graphic identity is that we don't have a name for it. It is just "our branding information" or, maybe, "the brand we got from Pentagram". That can be particularly challenging when talking about identity schemes from previous years, or comparing between two. I'd love for us to have an actual name to refer to this.

It was in that vein that I chose `tesseract` for this - it isn't perfect, but the network of lines does bring to mind (at least for me) a tesseract.

(All this said, I understand that this PR isn't the venue to try and settle any questions about what to call our identity package - I'll update this, probably to `app-theme`, in a future commit)